docs/introduction/03-filter-el.ts

```diff
@@ -5,6 +5,9 @@ const crawler = new CheerioCrawler({
     async requestHandler({ $, request, enqueueLinks }) {
         const title = $('title').text();
         console.log(`The title of "${request.url}" is: ${title}.`);
+        // The default behavior of enqueueLinks is to stay on the same hostname,
+        // so it does not require any parameters.
+        // This does not include subdomains.
```

Comment:
```suggestion
        // This will ensure the subdomain stays the same.
```

```md
Protocol          Domain
┌────┐          ┌─────────┐
https://example.crawlee.dev/...
│       └─────────────────┤
│             Hostname    │
│                         │
└─────────────────────────┘
         Origin
```